roles/seqreports/tasks/main.yml

```diff
@@ -25,9 +25,6 @@
 
 - import_tasks: pre_reqs.yml
 
-- import_tasks: seqreports_dev.yml
-  when: not miarka_deployment | bool
```

Comment:
I'm curious about the 'miarka_deployment' variable. Could you explain how it is used? In what contexts except miarka would this be deployed?

Reply:
In short, I plan to remove miarka_deployment variable in future PR. This miarka_deployment variable is something that I on a second thought want to remove, because it's hard to understand unless one is into the seqreports-dev role. Perhaps it's bettor to explain in person, but I give it a go here... The seqreports role (in miarka-provision) is called from sysman, when provisioning the seqreports-dev machine, that we hosts within snpseq. This is possible by including miarka-provision as a git submodule to the sysman repo. (git submodule: miarka-provision is provided as a subdirectory within sysman, and the ansible.cfg is updated to read roles from this subdirectory as well). It turned out that there were small and yet irritating differences between seqreports-dev and seqreports on miarka, regarding the "mode" attribute. I plan to change the current implementation of this, so that "mode" is configured in the same manner in seqreports-dev as on miarka, i.e. do all changes at the end recursively.

Reply:
Please tell me if you want me to explain in person instead...

Reply:
Ok I think I understand. My only feedback in this case would be that there should be a README that describes all of this, to give some context to a person looking at this repo without knowledge that system management exists. This does not have to be in this pull request, though. Otherwise approved.